browser.js

```diff
@@ -26,17 +26,19 @@ Updates should happen there first.
 */
 const scrollToTweet = event => { // eslint-disable-line
 	const $ = document.querySelector.bind(document);
-	const navHeight = $('nav').clientHeight;
 	const tweets = document.querySelectorAll('._222QxFjc[role="row"]');
 	const currentTop = window.scrollY;
 	const bufferTop = 5; // added because somtimes the scrolling is off by a few px
 	const keyCode = event.charCode; // TODO: replace with key when Chrome 51 is out https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
 
+	// calculate the height of the navigation/header to use for scrolling
+	const calculatedNavHeight = () => $('header').clientHeight + ($('header').clientHeight - $('nav').clientHeight);
```

Comment:
Why use a function at all here? Seems like a normal variable assignment would be enough.